packages/opds-web-client/src/utils/file.ts

```diff
@@ -13,6 +13,14 @@ export const typeMap: Record<MediaType, { extension: string; name: string }> = {
     extension: ".epub",
     name: "EPUB"
   },
+  "application/vnd.librarysimplified.web-epub": {
+    extension: ".xml",
+    name: "WEB EPUB"
```

Comment:
I think as we add new items we should try to make these as "human friendly" as possible, since they will be displayed. Is this the way we would want to display these books to the user?